frontend/src/scenes/experiments/ExperimentMetricForm.tsx

```diff
@@ -14,6 +15,25 @@ import {
     metricToQuery,
 } from './utils'
 
+const dataWarehousePopoverFields: DataWarehousePopoverField[] = [
+    {
+        key: 'timestamp_field',
+        label: 'Timestamp Field',
+    },
+    {
+        key: 'after_exposure_identifier_field',
+        label: 'Data Warehouse Identifier Field',
```

Comment:
I have very little context on this - but this doesn't really mean anything to me as a user. `Data Warehouse` isn't really a term that's used in the product at the moment, so I wouldn't really know what to select here. Likewise with the `Event Identifier Field` below

Reply:
Yeah, I agree. I don't feel particularly inspired right now, though.